ppr-ui/src/components/common/HelpInformationToggle.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <SimpleHelpToggle
```

Comment:
Yes naming is hard.

Maybe I can rename it to `ContactRegisteriesToggle`

I do want to highlight that it is a toggle, but I can see how HelpInformationToggle and SimpleHelpToggle can be confused.

I am not so worried about toggle within a toggle, because its just an instance of a toggle leveraging previous toggle functionality.

Reply:
`SimpleHelpToggle` already allows rendering the content (via default slot) so maybe we can just use that, instead of wrapping the component with another component? `HelpInformationToggle` doesn't really extend the functionality of `SimpleHelpToggle`. You already used it in a similar way, in `YourHome` component.

Reply:
Renamed component and added comment to help differentiate.